src/services/apiService.js

```diff
@@ -217,9 +217,11 @@ angular.module('openshiftCommon')
       if (!_.get(APIS_CFG, ["groups", resource.group, "versions", resource.version, "resources", primaryResource])) {
         return undefined;
       }
+      var hostPrefixObj = _.get(APIS_CFG, ["groups", resource.group, 'hostPrefix']) || APIS_CFG;
```

Comment:
That actually only works when get finds an undefined, in this case it gets a null.  I double checked the docs because I was surprised by it.